nodemon/egg-pterodactyl-nodemon.json

```
@@ -10,17 +10,17 @@
     "description": "Nodemon javascript egg that automatically restarts the node application when file changes in the directory are detected\r\n\r\nThis will clone a git repo for a bot. it defaults to master if no branch is specified.\r\n\r\nInstalls the node_modules on install. If you set user_upload, then I assume you know what you are doing.",
     "features": null,
     "docker_images": {
-        "ghcr.io/parkervcp/yolks:nodejs_12": "ghcr.io/parkervcp/yolks:nodejs_12",
-        "ghcr.io/parkervcp/yolks:nodejs_14": "ghcr.io/parkervcp/yolks:nodejs_14",
-        "ghcr.io/parkervcp/yolks:nodejs_15": "ghcr.io/parkervcp/yolks:nodejs_15",
-        "ghcr.io/parkervcp/yolks:nodejs_16": "ghcr.io/parkervcp/yolks:nodejs_16",
-        "ghcr.io/parkervcp/yolks:nodejs_17": "ghcr.io/parkervcp/yolks:nodejs_17",
-        "ghcr.io/parkervcp/yolks:nodejs_18": "ghcr.io/parkervcp/yolks:nodejs_18",
-        "ghcr.io/parkervcp/yolks:nodejs_19": "ghcr.io/parkervcp/yolks:nodejs_19",
-        "ghcr.io/parkervcp/yolks:nodejs_20": "ghcr.io/parkervcp/yolks:nodejs_20",
-        "ghcr.io/parkervcp/yolks:nodejs_21": "ghcr.io/parkervcp/yolks:nodejs_21",
-        "ghcr.io/parkervcp/yolks:nodejs_22": "ghcr.io/parkervcp/yolks:nodejs_22",
-        "ghcr.io/parkervcp/yolks:nodejs_23": "ghcr.io/parkervcp/yolks:nodejs_23"
+        "Nodejs 24": "ghcr.io\/parkervcp\/yolks:nodejs_24",
+        "Nodejs 23": "ghcr.io\/parkervcp\/yolks:nodejs_23",
+        "Nodejs 22": "ghcr.io\/parkervcp\/yolks:nodejs_22",
+        "Nodejs 21": "ghcr.io\/parkervcp\/yolks:nodejs_21",
+        "Nodejs 20": "ghcr.io\/parkervcp\/yolks:nodejs_20",
+        "Nodejs 19": "ghcr.io\/parkervcp\/yolks:nodejs_19",
+        "Nodejs 18": "ghcr.io\/parkervcp\/yolks:nodejs_18",
+        "Nodejs 17": "ghcr.io\/parkervcp\/yolks:nodejs_17",
+        "Nodejs 16": "ghcr.io\/parkervcp\/yolks:nodejs_16",
+        "Nodejs 14": "ghcr.io\/parkervcp\/yolks:nodejs_14",
+        "Nodejs 12": "ghcr.io\/parkervcp\/yolks:nodejs_12"
```

```suggestion
        "Nodejs 12": "ghcr.io/parkervcp/yolks:nodejs_12",
        "Nodejs 14": "ghcr.io/parkervcp/yolks:nodejs_14",
        "Nodejs 16": "ghcr.io/parkervcp/yolks:nodejs_16",
        "Nodejs 17": "ghcr.io/parkervcp/yolks:nodejs_17",
        "Nodejs 18": "ghcr.io/parkervcp/yolks:nodejs_18",
        "Nodejs 19": "ghcr.io/parkervcp/yolks:nodejs_19",
        "Nodejs 20": "ghcr.io/parkervcp/yolks:nodejs_20",
        "Nodejs 21": "ghcr.io/parkervcp/yolks:nodejs_21",
        "Nodejs 22": "ghcr.io/parkervcp/yolks:nodejs_22",
        "Nodejs 23": "ghcr.io/parkervcp/yolks:nodejs_23",
        "Nodejs 24": "ghcr.io/parkervcp/yolks:nodejs_24"
```
Pterodactyl sorts them differently